### PR TITLE
Add Nerfels and SDF-based RGBD tracking

### PR DESCRIPTION
**Pull request by jiadingfang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@ If you find this repository useful, please consider [citing](#citation) and STAR
 
 * **NeRF-Pose**: " A First-Reconstruct-Then-Regress Approach for Weakly-supervised 6D Object Pose Estimation", *arXiv*. [[Paper](https://arxiv.org/pdf/2203.04802.pdf)]
 
+* **Nerfels**: "Renderable Neural Codes for Improved Camera Pose Estimation", *arXiv*. [[Paper](https://arxiv.org/pdf/2206.01916.pdf)]
+
+* "SDF-based RGB-D Camera Tracking in Neural Scene Representations", *ICRA Workshop, 2022*. [[Paper](https://neural-implicit-workshop.stanford.edu/assets/pdf/bruns.pdf)]
```
Comment (zubair-irshad) [inline README.md:27]: I think this should be under SLAM. Could you please update?

**Review by zubair-irshad — CHANGES_REQUESTED** [review]
I think this should be under SLAM. Could you please update?

* "SDF-based RGB-D Camera Tracking in Neural Scene Representations", *ICRA Workshop, 2022*. [[Paper](https://neural-implicit-workshop.stanford.edu/assets/pdf/bruns.pdf)]

**Comment by jiadingfang** [comment]
To me, SLAM means the algorithm needs to figure out both pose and map at the same time (preferably in real time). Both of these two works assume pre-computed implicit representations and focus on the pose optimization. Similarly, LENS should be moved to "pose estimation". 

On another note, iSDF (at its current state) may not qualify for SLAM either because they actually use gt poses (I talked the author, he said they tried to learn poses at the same time, but not successful). 

Let me know what you think.

**Comment by zubair-irshad** [comment]
Makes sense, thanks for the clarification. Maybe move them to their separate own thread titled 'Camera Pose optimization/Tracking' or a related name? 

I thought earlier about merging 6D object pose estimation and camera pose estimation/optimization; I think robotic folks like to have a separate thread/line of works for 6D Object Pose Estimation including instance-based/category-level works etc. 

Let me know what you think!

**Comment by jiadingfang** [comment]
What's the difference between _object_ and _camera_ pose optimization? Aren't they just inverse to each other?

**Comment by zubair-irshad** [comment]
For a single object like [SRN dataset](https://github.com/vsitzmann/scene-representation-networks#data), yes this seems to be true (there are inverse to each other). For multi-object scenarios, the camera pose remains same but each object's rotation and translation w.r.t the camera changes (also used to fit a tighter bounding box around object of interest). Particularly evident for robotics datasets and dynamic moving objects in case of cars for Kitti. Here is a tutorial for [6D object pose](https://haosulab.github.io/ml-meets-geometry/WI21/Lectures/L12_6D_Pose_Estimation_II.pdf). 

[Nerf-Pose](https://arxiv.org/pdf/2203.04802.pdf) is trying to solve the object-pose estimation problem given GT camera poses whereas [Nerfels](https://arxiv.org/pdf/2206.01916.pdf) for camera pose. IMHO, Nerfels and other could fit under both categories although could be useful if we create a separate thread for each?

**Comment by zubair-irshad** [comment]
Closing this. Please see this [PR](https://github.com/zubair-irshad/Awesome-Implicit-NeRF-Robotics/commit/32f12a1ab4428053ada8ee01383fab426901e824). SLAM currently is loosely defined to incorporate camera pose optimization/tracking works but feel free to open sub-threads in future to expand this or move this sub-work into it's own thread. Thanks for your contribution

**Pull request closed without merging**